### PR TITLE
Fb issues list def test conversion

### PR DESCRIPTION
**Pull request by cnathe**
#### Rationale
With the issues list def designer conversion complete and the experimental feature flag being removed in the related platform branch, we need to update any existing automated test usages of the GWT issues list designer.

#### Related Pull Requests
* https://github.com/LabKey/assayRequest/pull/27
* https://github.com/LabKey/biologics/pull/588
* https://github.com/LabKey/platform/pull/1184
* https://github.com/LabKey/labkey-ui-components/pull/252

#### Changes
* Update IssuesAdminPage for new UI properties panel and to override the DomainDesigner
* Convert existing test usages of IssuesAdminPage to use new helpers
* Add IssuesAttachmentTest to the Issues category on TeamCity

### CODE REVIEW

**Review by labkey-tchad — COMMENTED** [review]
While we're touching this, could you rename `AdminPage` to `IssuesAdminPage`. I regret giving it such a generic name.

**Comment by cnathe** [comment]
> While we're touching this, could you rename `AdminPage` to `IssuesAdminPage`. I regret giving it such a generic name.

Done.

File: src/org/labkey/test/pages/issues/IssuesAdminPage.java [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+import org.labkey.test.components.html.Input;
+import org.labkey.test.components.html.OptionSelect;
+import org.labkey.test.util.Maps;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import static org.labkey.test.WebDriverWrapper.WAIT_FOR_PAGE;
+import static org.labkey.test.WebDriverWrapper.waitFor;
+
+public class IssuesAdminPage extends DomainDesigner<IssuesAdminPage.ElementCache>
```
Comment (labkey-tchad) [inline src/org/labkey/test/pages/issues/IssuesAdminPage.java:37]: Add a pointer to the product code this class automates. Similar to `EditListDefinitionPage`.

File: src/org/labkey/test/components/IssueListDefDataRegion.java [hunk 2]
```diff
@@ -15,7 +15,7 @@
  */
 package org.labkey.test.components;
```
Comment (labkey-tchad) [inline src/org/labkey/test/components/IssueListDefDataRegion.java]: While we're touching these files, move this class to a more specific package.
```suggestion
package org.labkey.test.components.issues;
```

File: src/org/labkey/test/tests/issues/IssuesAttachmentTest.java [hunk 3]
```diff
@@ -41,7 +42,7 @@
 
 import static org.junit.Assert.assertFalse;
 
-@Category({DailyC.class})
+@Category({Issues.class, DailyC.class})
```
Comment (labkey-tchad) [inline src/org/labkey/test/tests/issues/IssuesAttachmentTest.java:45]: This is unnecessary but harmless. The test runner will add this test to the issues suite based on the package name (`org.labkey.test.tests.issues`).

**Review by labkey-tchad — APPROVED** [review]

File: src/org/labkey/test/tests/issues/IssuesAttachmentTest.java (hunk 3 above)
Comment (cnathe) [reply src/org/labkey/test/tests/issues/IssuesAttachmentTest.java:45]: Ah cool. I didn't know that. I was using the category annotations as a way to find out what existing automated tests needed to be converted. That is one of the reasons I added it here.

**Review by labkey-chrisj — APPROVED** [review]

File: src/org/labkey/test/pages/issues/IssuesAdminPage.java (hunk 1 above)
Comment (cnathe) [reply src/org/labkey/test/pages/issues/IssuesAdminPage.java:37]: Done

File: src/org/labkey/test/components/IssueListDefDataRegion.java (hunk 2 above)
Comment (cnathe) [reply src/org/labkey/test/components/IssueListDefDataRegion.java]: Done

**Pull request merged**